dev-guide/kubernetes_dashboard.md

```diff
@@ -0,0 +1,17 @@
+Kubernetes dashboard
+====================
+
+The NAIS platform comes with the default [Kubernetes dashboard](https://github.com/kubernetes/dashboard) running in it's own namespace in each cluster. It's also running without an ingress, so you need to use [port-forward](https://kubernetes.io/docs/tasks/access-application-cluster/port-forward-access-application-cluster/) to interact with it.
+
+In [Kubectl](https://kubernetes.io/docs/reference/kubectl/kubectl/) 1.10 and later you can run the following command:
+
+```
+kubectl port-forward deployment/kubernetes-dashboard 9090:9090 --namespace kubernetes-dashboard
+```
+
+In earlier Kubectl versions, you need to specify the pod that you are targeting with the `port-forward`:
```

Comment:
We might not want users to know about old Kubernetes versions? Isn't that an internal concern?

Reply:
Både ja og nei, men enig med deg at vi kanskje bare burde forklare siste versjon. Og så heller be folk oppdatere når de lurer på hvorfor ting feiler i #nais.